DocSum/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -39,67 +68,66 @@ docker build -t opea/docsum:latest --build-arg https_proxy=$https_proxy --build-
 Build the frontend Docker image via below command:
 
 ```bash
-cd GenAIExamples/DocSum/ui
-docker build -t opea/docsum-ui:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f docker/Dockerfile .
+cd GenAIExamples/DocSum/ui/docker
```

Comment:
The way it's written I am getting a build error. I think this needs to be changed back to navigating to the `ui` folder and then using `-f docker/Dockerfile.gradio`
```suggestion
cd GenAIExamples/DocSum/ui
```

My docker build error is:
```
 => ERROR [4/7] COPY ./gradio/docsum_ui_gradio.py /home/user/docsum_ui_gradio.py                                                                                                                                                                                                             0.0s
 => ERROR [5/7] COPY ./gradio/requirements.txt /home/user/requirements.txt                                                                                                                                                                                                                   0.0s
------
 > [4/7] COPY ./gradio/docsum_ui_gradio.py /home/user/docsum_ui_gradio.py:
------
------
 > [5/7] COPY ./gradio/requirements.txt /home/user/requirements.txt:
------
Dockerfile.gradio:21
--------------------
  19 |     # Copy the application code and requirements file to the container
  20 |     COPY ./gradio/docsum_ui_gradio.py /home/user/docsum_ui_gradio.py
  21 | >>> COPY ./gradio/requirements.txt /home/user/requirements.txt 
  22 |     
  23 |     # Install Python dependencies
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref e4041a4d-34f1-4db4-a5ee-a6cb16701b70::z6z1y6myra817r1nrl9p15cvs: "/gradio/requirements.txt": not found
```